doc/stdenv.xml

```diff
@@ -1191,17 +1191,35 @@ echo @foo@
     <replaceable>file</replaceable>.</para></listitem>
   </varlistentry>
 
+  <varlistentry xml:id='fun-baseHash'>
+    <term>
+      <function>baseHash</function>
+      <replaceable>path</replaceable>
+      <replaceable>suffix</replaceable>
+    </term>
+    <listitem><para>Strips the directory and hash part of a store
+    path, storing the name part in the environment variable
+    <literal>strippedName</literal>. If <literal>suffix</literal> is also
```

Comment:
Fixed in 9890b594c78c3e256123a6f4a688fdecd107e382. The doc is also missing the advanced usage that's documented directly in the source but I don't know how to express that in XML.